apps/api/src/app/inbox/usecases/session/session.usecase.ts

```diff
@@ -161,4 +210,421 @@ export class Session {
+                label: {
+                  type: 'string',
+                },
+                redirect: {
+                  type: 'object',
+                  properties: {
+                    url: {
+                      type: 'string',
+                      pattern:
+                        '^(?:\\{\\{[^}]*\\}\\}.*|(?!mailto:)(?:https?:\\/\\/[^\\s/$.?#][^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)|\\/[^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)$',
+                    },
+                    target: {
+                      type: 'string',
+                      enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'],
+                    },
+                  },
+                  required: ['url', 'target'],
+                  additionalProperties: false,
+                },
+              },
+              required: ['label'],
+              additionalProperties: false,
+            },
+            secondaryAction: {
+              type: 'object',
+              properties: {
+                label: {
+                  type: 'string',
+                },
+                redirect: {
+                  type: 'object',
+                  properties: {
+                    url: {
+                      type: 'string',
+                      pattern:
+                        '^(?:\\{\\{[^}]*\\}\\}.*|(?!mailto:)(?:https?:\\/\\/[^\\s/$.?#][^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)|\\/[^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)$',
+                    },
+                    target: {
+                      type: 'string',
+                      enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'],
+                    },
+                  },
+                  required: ['url', 'target'],
+                  additionalProperties: false,
+                },
+              },
+              required: ['label'],
+              additionalProperties: false,
+            },
+            data: {
+              type: 'object',
+            },
+            redirect: {
+              type: 'object',
+              properties: {
+                url: {
+                  type: 'string',
+                  pattern:
+                    '^(?:\\{\\{[^}]*\\}\\}.*|(?!mailto:)(?:https?:\\/\\/[^\\s/$.?#][^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)|\\/[^\\s]*(?:\\{\\{[^}]*\\}\\})*[^\\s]*)$',
+                },
+                target: {
+                  type: 'string',
+                  enum: ['_self', '_blank', '_parent', '_top', '_unfencedTop'],
+                },
+              },
+              required: ['url', 'target'],
+              additionalProperties: false,
+            },
+          },
+          additionalProperties: false,
+        },
+        uiSchema: {
+          group: 'IN_APP',
+          properties: {
+            body: {
+              component: 'IN_APP_BODY',
+              placeholder: '',
+            },
+            avatar: {
+              component: 'IN_APP_AVATAR',
+              placeholder: 'https://dashboard-v2.novu.co/images/info.svg',
```

Comment:
```suggestion
              placeholder: 'https://dashboard.novu.co/images/info.svg',
```

Ideally we will be depreacting soon the old domain, so if you can update it